sloop_ros/Loc loc
# free, obstacle or unnknown
string type
# a string label
string label